hikari/internal/routes.py

```diff
@@ -597,51 +619,69 @@ def compile_to_file(
 GET_GATEWAY: typing.Final[Route] = Route(GET, "/gateway")
 GET_GATEWAY_BOT: typing.Final[Route] = Route(GET, "/gateway/bot")
 
-PNG: typing.Final[str] = "png"
-JPEG_JPG: typing.Final[tuple[str, str]] = ("jpeg", "jpg")
-WEBP: typing.Final[str] = "webp"
-GIF: typing.Final[str] = "gif"
-LOTTIE: typing.Final[str] = "json"  # https://airbnb.io/lottie/
+PNG: typing.Final[str] = "PNG"
+JPEG: typing.Final[str] = "JPEG"
+JPG: typing.Final[str] = "JPG"
+WEBP: typing.Final[str] = "WEBP"
+APNG: typing.Final[str] = "APNG"
+AWEBP: typing.Final[str] = "AWEBP"
+GIF: typing.Final[str] = "GIF"
+LOTTIE: typing.Final[str] = "LOTTIE"  # https://airbnb.io/lottie/
+
+CDN_FORMAT_EXT: typing.Final[dict[str, str]] = {
+    PNG: "png",
+    JPEG: "jpeg",
+    JPG: "jpg",
+    WEBP: "webp",
+    APNG: "png",
+    AWEBP: "webp",
+    GIF: "gif",
+    LOTTIE: "json",
+}
```

Comment:
I am quite meh about this.

I know this is kinda going a bit back to the beginning, but would it be better to keep it mostly how it was (with the `ext` parameter) and for AWEBP support, just allow passing `animated=True` and then the webp returned will be animated.

Mostly thinking this because the complexity added by this pull request, specially when handling and keeping things in mind is quite confusing, and I believe could be more straight forward.